torch/nn/modules/rnn.py

```diff
@@ -455,6 +460,100 @@ class LSTM(RNNBase):
 
     def __init__(self, *args, **kwargs):
         super(LSTM, self).__init__('LSTM', *args, **kwargs)
+        torch._jit_internal.overload(type(self), 'forward', (self.forward_tensor, self.forward_packed))
+
+    @weak_script_method
+    def check_hidden_size(self, hx, expected_hidden_size, msg='Expected hidden size {}, got {}'):
```

Comment:
The weak methods below were copied from RNNBase and specialized for LSTM since `hx` is a tuple for `LSTM` only and not the other `rnn` modules.